### ISSUE TITLE
v. 3.16.1 no sount on Win 11

### ISSUE THREAD

**Issue by capkwantum** [HenriWahl/Nagstamon#1068]
Just upgraded to 3.16.1 and I noticed that no sound is played (I tried the default ones also)

3.14.0 works fine, 3.16.0 and 3.16.1 both have this issue

Any ideas?

Thanks in advance!

**Comment by HenriWahl** [comment]
Interesting... what a mess. Yes, neither on Windows 10 or Windows 11 there is any sound played :-( - Can you try the debug version of Nagstamon https://github.com/HenriWahl/Nagstamon/releases/download/v3.16.1/Nagstamon-3.16.1-win64_debug.zip, just to get some output?

**Comment by HenriWahl** [comment]
I rebuilt the Installer with an older version of Qt6, which still plays sound. Works for me now. Can you please test from https://github.com/HenriWahl/Nagstamon/releases/tag/latest ?

And BTW, can you please test dark mode in Windows 11 with this release?

**Comment by damienkent** [comment]
> I rebuilt the Installer with an older version of Qt6, which still plays sound. Works for me now. Can you please test from https://github.com/HenriWahl/Nagstamon/releases/tag/latest ?
> 
> And BTW, can you please test dark mode in Windows 11 with this release?

Confirmed: sound working on Windows 11 23H2 (Build 22631.3880) with Nagstamon version 3.17-20241018.
However, dark mode does not appear to work as it did in Nagstamon 3.16.1. (Same windows version and build).

**Comment by HenriWahl** [comment]
What about the very latest from some minutes ago from https://github.com/HenriWahl/Nagstamon/releases/tag/latest ? Regarding sound and dark mode?

**Comment by damienkent** [comment]
> What about the very latest from some minutes ago from https://github.com/HenriWahl/Nagstamon/releases/tag/latest ? Regarding sound and dark mode?

Tested again with 3.17-20241023.
The sound is still working, but there is still no dark mode.

**Comment by HenriWahl** [comment]
Maybe the sound is more important than dark mode so we need to wait for a Qt6 version which handles both.

**Comment by damienkent** [comment]
Agreed. As much as I enjoy reducing the amount my retina burns I think I prefer to be notified audibly when my SAN goes down. 😜
